### PR TITLE
Add support for browserConfig and update examples

### PR DESCRIPTION
**Pull request by boatengyeboah**
Adds `browserConfig` to allow setting all puppeteer browser options instead of just `isHeadless`. Also updated the examples.

### CODE REVIEW

File: examples/web/twitch.browse [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # Pass `--web` to browse when running this example
 # $ browse --web ./examples/twitch.browse
 
-set headless false
+browserConfig {
+  set headless true
```
Comment (pranaygp) [inline examples/web/twitch.browse:5]: `false` ?

File: packages/web/lib/page.js [hunk 2]
```diff
@@ -110,7 +110,7 @@ const getPageScope = (parent) => {
       });
       return null;
     },
-    config: (_) => (_) => async (ruleSet) => {
+    pageConfig: (_) => (_) => async (ruleSet) => {
```
Comment (pranaygp) [inline packages/web/lib/page.js:113]: we use `config` in a few examples (like examples/web/thegazette.browse). Let's update them to use `pageConfig`

**Comment by pranaygp** [comment]
Good PR. I don't want to merge anything new to master right now since we're awaiting repl.it hackathon results, so I'll create a new branch and release channel so you can start using this at YSplit this week @boatengyeboah

File: examples/web/twitch.browse (hunk 1 above)
Comment (boatengyeboah) [reply examples/web/twitch.browse]: Oops, fixed it.

**Pull request merged**